add new templates for azure data share account

dadihe opened the pull request:
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)


- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

bmoore-msft commented on 101-data-share-account/README.md:
```diff
@@ -0,0 +1,24 @@
+# Copy data from one folder to another folder in an Azure Blob Storage
```
this doesn't look correct...

dadihe replied:
fixed

bmoore-msft commented on 101-data-share-account/README.md:
```diff
@@ -0,0 +1,24 @@
+
+[![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-data-share-account%2Fazuredeploy.json)
+
+When you deploy this Azure Resource Manager template, a data share account is created.
```
what is a data share account?  is there a doc link?

dadihe replied:
https://docs.microsoft.com/en-us/azure/data-share/overview.  I've add this link to the descriptions. should I apply this anywhere else?

dadihe replied:
> https://docs.microsoft.com/en-us/azure/data-share/overview. I've add this link to the descriptions. should I apply this anywhere else?



> what is a data share account? is there a doc link?

https://docs.microsoft.com/en-us/azure/data-share/overview. I've add this link to the descriptions. should I apply this anywhere else?

dadihe replied:
https://docs.microsoft.com/en-us/azure/data-share/overview. I've add this link to the descriptions. should I apply this anywhere else?

bmoore-msft commented on 101-data-share-account/azuredeploy.json:
```diff
@@ -0,0 +1,34 @@
+            "apiVersion": "2019-11-01",
+            "name": "[parameters('accounts_template_name')]",
+            "location": "[parameters('location')]",
+            "identity": {
```
how is this used?  also, the property set doesn't look correct, on a systemAssigned msi the tenant and principal are created by the system, not supplied by the user

dadihe replied:
removed systemassigned msi. user will enter only account name and location to deploy the template.

bmoore-msft commented on 101-data-share-account/metadata.json:
```diff
@@ -0,0 +1,22 @@
+  "summary": "Create a Data Share account",
+  "githubUsername": "dadihe",
+  "dateUpdated": "2020-06-29",
+  "environments":[
```
this isn't correct, see: https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE#environments

dadihe replied:
updated with default value.

dadihe replied:
It fails build bc' it is using westus to deploy where we currently don't support. Should I update these issue somewhere? also we have 6 other folders/resource to be updated, can I include them into this PR all should I separate them one each? Thanks

dadihe commented:
@bmoore-msft Hi Brian, as we discussed I've added a separate parameter file for usgov and corrected inappropriate descriptions. Can you please review again, thanks~
